src/javascript/app/pages/trade/analysis.js

```diff
@@ -73,10 +81,24 @@ const TradingAnalysis = (() => {
                 showChart();
             } else if (current_tab === 'tab_last_digit') {
                 const el_digit_underlying = $('#digit_underlying');
-                const underlying = $('#underlying option:selected').val();
+                const underlying = $('#underlying').val();
                 const tick       = $('#tick_count').val() || 100;
+
                 if (underlying !== el_digit_underlying.val() && el_digit_underlying.val() !== null ) {
                     el_digit_underlying.find(`option[value="${underlying}"]`).prop('selected', true).trigger('change');
+                    const digit_underlying_dropdown = el_digit_underlying.next('div.select-dropdown');
```

Comment:
is this jquery? if yes, please add `$` to it, like `const $digit_underlying_dropdown`